Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,162 @@
+using System.Collections.Generic;
+using Enums;
+using Models;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    public static GameManager Instance { get; private set; }
+    public User Cpu1 { get; private set; }
+    public User Cpu2 { get; private set; }
+    public User Player { get; private set; }
+
+    private void Awake()
+    {
+        Instance = this;
+
+        Cpu1 = new User();
+        Cpu2 = new User();
+        Player = new User();
+        
+        SetPointTables();
+        FillInventory();
+    }
+
+    private void SetPointTables()
+    {
+        Player.PointTable = new Dictionary<(InventoryItems, User), int>

```

Comment:
Why are the points assigned based on inventory items? Shouldn't this be on the structures itself?

Reply:
Because then you would need to add extra steps to the algorithm while 1 inventory item directly translates to 1 building, so it doesn’t really matter.